### PR TITLE
Remove Trustzone root enclave

### PR DESCRIPTION
**Pull request by dreemkiller**
Removed the Trustzone root enclave. This is part of the work on #165

### CODE REVIEW

**Comment by dominic-mulligan-arm** [comment]
@dreemkiller could you register your GPG key with Github then sign the commit in this PR and force push?  Unfortunately, this process needs to be done for every PR that was opened before I turned signing on.

**Comment by dreemkiller** [comment]
I have attempted to sign this one with GPG. Let me know how it goes...

**Comment by dreemkiller** [comment]
This is ready to go in if someone wants to review it

**Review by ShaleXIONG — APPROVED** [review]

File: runtime-manager/Cargo.toml [hunk 1]
```diff
@@ -131,6 +132,7 @@ wasmi = { git = "https://github.com/veracruz-project/wasmi.git", branch = "verac
 
 [build-dependencies]
 uuid = { version = "0.7", features = ["v4"] }
+libloading = "=0.5.2"
```
Comment (dominic-mulligan-arm) [inline runtime-manager/Cargo.toml:135]: nit: can you move `libloading` above `uuid` to keep alphabetical order of imports after the recent formatting?

File: test-collateral/generate-policy/Cargo.toml [hunk 2]
```diff
@@ -16,6 +16,7 @@ http = "=0.2.4"
 log = "=0.4.13"
 policy-utils = { path = "../../policy-utils", features = ["std"] }
 ring = { git = "https://github.com/veracruz-project/ring.git", branch = "veracruz", features = ["non_sgx"] }
+veracruz-utils = {path = "../../veracruz-utils", features = ["std", "tz"]}
```
Comment (dominic-mulligan-arm) [inline test-collateral/generate-policy/Cargo.toml:19]: nit: can you move `veracruz-utils` down to retain alphabetical ordering of the imports?

**Review by dominic-mulligan-arm — CHANGES_REQUESTED** [review]

**Review by dominic-mulligan-arm — APPROVED** [review]

**Comment by dominic-mulligan-arm** [comment]
+1 reached, merging.

**Pull request merged**